ufm-test-services/metamask/Dockerfile.local

```diff
@@ -0,0 +1,25 @@
+# Using the Playwright image
+FROM mcr.microsoft.com/playwright:v1.37.1-jammy
+
+# Setting the working directory
+WORKDIR /app
+
+# Update PATH
+ENV PATH /app/node_modules/.bin:$PATH
+
+RUN if [ "$METAMASK_RUN_HEADLESS" = "true" ]; then \
+        apt-get update && \
+        apt-get install -y xvfb && \
+        rm -rf /var/lib/apt/lists/* ; \
+    fi
```

Comment:
I like where your head is at here but it seems unnecessary. In practice this just ends up getting cached so there is no real reason to skip this